com.github.wwmm.easyeffects.yml

```
@@ -0,0 +1,389 @@
+# see https://github.com/wwmm/easyeffects/blob/master/PKGBUILD for dependencies
+# If you get errors like 137, you might need pass --jobs=1 to flatpak-builder to limit cpu threads.
+  # discussion at here https://github.com/flathub/com.github.wwmm.pulseeffects/pull/53
+# todo
+# add x-checker-data
+# fix icon
+
+id: com.github.wwmm.easyeffects
+runtime: org.gnome.Platform
+runtime-version: '40'
+sdk: org.gnome.Sdk
+command: easyeffects
+rename-icon: easyeffects
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  - --filesystem=xdg-run/pipewire-0:ro
+    # LADSPA Plugin PATH
+  - --env=LADSPA_PATH=/app/lib/ladspa:/app/extensions/Plugins/ladspa
+    # LV2 Plugin PATH
+  - --env=LV2_PATH=/app/lib/lv2:/app/extensions/Plugins/lv2
+cleanup:
+  - '*.a'
+  - '*.h'
+  - '*.la'
+  - /bin/analyseplugin
+  - /bin/applyplugin
+  - /bin/listplugins
+  - /include
+  - /lib/pkgconfig
+  - /lib/python*
+  - /share/info
+
+add-extensions:
+  org.freedesktop.LinuxAudio.Plugins:
+    directory: extensions/Plugins
+    version: "20.08"
+    add-ld-path: lib
+    merge-dirs: ladspa;dssi;lv2;lxvst;vst3
```

```suggestion
    merge-dirs: ladspa;lv2
```

Since it only support these.